deepchecks/checks/integrity/mixed_data_types.py

```diff
@@ -9,25 +9,25 @@
 # ----------------------------------------------------------------------------
 #
 """module contains Mixed Types check."""
-from typing import List, Union
+from typing import List, Union, Tuple
 import pandas as pd
 
 import numpy as np
 
 from deepchecks import Dataset
-from deepchecks.base.check import CheckResult, SingleDatasetBaseCheck, ConditionResult
+from deepchecks.base.check import CheckResult, SingleDatasetBaseCheck, ConditionResult, ConditionCategory
 from deepchecks.utils.dataframes import select_from_dataframe
 from deepchecks.utils.features import calculate_feature_importance_or_null, column_importance_sorter_df
 from deepchecks.utils.strings import is_string_column, format_percent, format_columns_for_condition
 from deepchecks.utils.validation import ensure_dataframe_type
 from deepchecks.utils.typing import Hashable
 
 
-__all__ = ['MixedTypes']
+__all__ = ['MixedDataTypes']
 
 
-class MixedTypes(SingleDatasetBaseCheck):
-    """Search for various types of data in (a) column[s], including hidden mixes in strings.
+class MixedDataTypes(SingleDatasetBaseCheck):
+    """Detect a minority of a data type within a column, such as few string samples in a mostly numeric column.
```

Comment:
Minority is a bit weird as its not common to use in these things, IMO.
What about an anomaly?